src/runtime_src/core/common/CMakeLists.txt

```diff
@@ -16,6 +16,9 @@ file(GLOB XRT_CORECOMMON_LIB_FILES
 # Files to include in object list
 file(GLOB XRT_CORECOMMON_OBJ_FILES
   "scheduler.*"
+  "dd.*"
```

Comment:
What's the reason dd sensor and utils files were added to this list?  Do they themselves depend on the coreutils library?   

Unfortunately dd* of can't be under core/common  because its linux specific, e.g. won't compile on windows.   It should be rewritten to free form command line parsing or rewritten to use boost::program_options.   If you can do this great, if not, then you have to copy dd* to edge.

sensor and utils should be in the corecommon_lib_files list.

Reply:
1. No, sensor and utils dont depend on coreutils library. Moved sensor and utils in the corecommon_lib_files list.
2. Actually we are not implementing dd in this phase. So for now we don't need to move dd to common. But point taken, i will implement dd with boost::program_options.